### PR TITLE
Experimental/reduce image size building npm node

### PR DESCRIPTION
**Pull request by cesarjorgemartinez**

### CODE REVIEW

**Comment by cesarjorgemartinez** [comment]
We need to pass all tests features and performances

File: doc/manuals/installation.md [hunk 1]
```diff
@@ -115,13 +115,37 @@ Last but not least, the STH process (a `node` process) runs the as `sth` user.
...
-On the other hand a [`Dockerfile`](https://github.com/telefonicaid/fiware-sth-comet/blob/master/Dockerfile) and a [`docker-compose.yml`](https://github.com/telefonicaid/fiware-sth-comet/blob/master/docker-compose.yml) files have also been included in the component repository in Github to quickly and easily start your own instance of the STH component, even including the needed associated MongoDB instance where all the data will be stored.
+On the other hand a [`Dockerfile`](https://github.com/telefonicaid/fiware-sth-comet/blob/master/docker/Dockerfile) and a [`docker-compose.yml`](https://github.com/telefonicaid/fiware-sth-comet/blob/master/docker/docker-compose.yml) files have also been included in the component repository in Github to quickly and easily start your own instance of the STH component, even including the needed associated MongoDB instance where all the data will be stored.
 
 To do it, follow the next steps once you have installed Docker in your machine:
 
 1. Navigate to the path where the component repository was cloned.
-2. Compose and run the STH component image:
+2. Compose and run the STH component image
+
+* In foreground mode
```
Comment (gtorodelvalle) [inline doc/manuals/installation.md]: Please, remove the blank line before this one and add 4 spaces for a proper indentation. Add ":" at the end, please.

File: doc/manuals/installation.md [hunk 2]
```diff
@@ -115,13 +115,37 @@ Last but not least, the STH process (a `node` process) runs the as `sth` user.
...
 ```bash
-cd fiware-sth-comet/
-docker-compose up
+sudo docker-compose -f docker/docker-compose.yml up
+```
+
+* Or in detached mode
```
Comment (gtorodelvalle) [inline doc/manuals/installation.md]: Please, remove the blank line before this one and add 4 spaces for a proper indentation. Add ":" at the end, please.

File: doc/manuals/installation.md [hunk 3]
```diff
@@ -115,13 +115,37 @@ Last but not least, the STH process (a `node` process) runs the as `sth` user.
...
 ```
+
+There is also the possibility to build your own local Docker image of the STH component.
+
+To do it, follow the next steps once you have installed Docker in your machine:
+
+1. Navigate to the path where the component repository was cloned.
+2. Launch a Docker build
+
+* With default nodejs v0.10.42 version
```
Comment (gtorodelvalle) [inline doc/manuals/installation.md]: Please, remove the blank line before this one and add 4 spaces for a proper indentation. Add ":" at the end, please.
Please, substitute the included text by: "Using the default NodeJS 0.10.42 version:" (with ":" at the end).

File: doc/manuals/installation.md [hunk 4]
```diff
@@ -115,13 +115,37 @@ Last but not least, the STH process (a `node` process) runs the as `sth` user.
...
+```bash
+sudo docker build -f docker/Dockerfile .
+```
+
+* With specific nodejs version
```
Comment (gtorodelvalle) [inline doc/manuals/installation.md]: Please, remove the blank line before this one and add 4 spaces for a proper indentation. Add ":" at the end, please.
Please, substitute the text by: "Using a specific NodeJS version (v0.10.46 in the next example):" (":" at the end)

**Comment by cesarjorgemartinez** [comment]
ERROR!

**Pull request closed without merging**